test/topology_custom/test_mv_large_inserts.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+from test.pylib.manager_client import ManagerClient
+
+import asyncio
+import pytest
+import time
+import logging
+from test.topology.conftest import skip_mode
+from cassandra.cqltypes import Int32Type
+
+logger = logging.getLogger(__name__)
+
+async def insert_with_concurrency(cql, concurrency):
+    def serialize_int(i):
+        return Int32Type.serialize(i, cql.cluster.protocol_version)
+    def get_replicas(key):
+        return cql.cluster.metadata.get_replicas("ks", serialize_int(key))
+    for key in range(100):
+        logger.info(f"Starting writes with concurrency {concurrency}")
+        async def do_inserts(i: int):
+            key = i
+            local_node = get_replicas(key)[0]
+            while local_node == get_replicas(i)[0]:
+                i = i + concurrency
+            await cql.run_async(f"INSERT INTO ks.tab (key, c, v) VALUES ({key}, {i}, '{10000 * "a"}')")
+
+        tasks = [asyncio.create_task(do_inserts(i)) for i in range(concurrency)]
+        await asyncio.gather(*tasks)
+        logger.info(f"Finished writes with concurrency {concurrency}")
+
+async def wait_for_views(cql, mvs_count, node_count):
+    deadline = time.time() + 120
+    while time.time() < deadline:
+        done = await cql.run_async(f"SELECT COUNT(*) FROM system_distributed.view_build_status WHERE status = 'SUCCESS' ALLOW FILTERING")
+        logger.info(f"Views built: {done[0][0]}")
+        if done[0][0] == node_count * mvs_count:
+            return
+        else:
+            time.sleep(0.2)
+    raise Exception("Timeout waiting for views to build")
+
+
+@pytest.mark.asyncio
+@skip_mode('release', "error injections aren't enabled in release mode")
+async def test_large_concurrency(manager: ManagerClient) -> None:
+    node_count = 2
+    await manager.servers_add(node_count, cmdline=['--error-injections-at-startup=view_update_limit'])
```

Comment:
I can't find this view_update_limit injection. Is it a patch you forgot to send? Or maybe I'm just searching incorrectly?

Reply:
Yes, it's part of https://github.com/scylladb/scylladb/pull/16819. I could send it separately but I hope https://github.com/scylladb/scylladb/pull/16819 will get merged soon